src/stylesheets/includes/_not-sorted-scss.scss

```diff
@@ -1396,4 +1396,16 @@
 
 .ui.flowing.popup {
   z-index: 10000;
+}
+
+.ui.active.dimmer {
+  z-index: 10000;
+}
+
+.ui.dimmer {
```

Comment:
- remove
- format file

Reply:
ui.dimmer removed. You do the scss formating